main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -85,7 +85,7 @@ object CompletionProvider {
     //
     val completions = source.flatMap(getContext(_, uri, pos)) match {
       case None => Nil
-      case Some(context) => getCompletions()(context, flix, index, root, deltaContext) ++ getCompletionsFromErrors(pos, currentErrors)(context, index, root)
+      case Some(context) => getCompletions()(context, flix, index, root.get, deltaContext) ++ getCompletionsFromErrors(pos, currentErrors)(context, index, root)
```

Comment:
I think the point is that we can a pattern match.

if there is no AST root then we return the empty list. Otherwise we actually call the getCompletions.